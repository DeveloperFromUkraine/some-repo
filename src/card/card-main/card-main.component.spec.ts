import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardMainComponent } from "./index";
import { ComponentTest } from '../../../test/test-bed/component';

describe('CardMainComponent', () => {
    let fixture: ComponentFixture<CardMainComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardMainComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardMainComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});