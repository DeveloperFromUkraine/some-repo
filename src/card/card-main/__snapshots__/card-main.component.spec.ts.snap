// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Main should match snapshot 1`] = `
<ign-card-main>
  
</ign-card-main>
`;
